src/Symfony/Component/EventDispatcher/Debug/WrappedListener.php

```diff
@@ -87,13 +88,13 @@ public function getPretty(): string
         return $this->pretty;
     }
 
-    public function getInfo(string $eventName): array
+    public function getInfo(string $eventName, int $priority = null): array
```

Comment:
adding an argument is a BC break
we should instead use `@param int|null $priority` and use func_get_arg (see existing similar BC layer on 5.4 I guess)

Reply:
Done